gce-containers-startup/utils/volumes.go

```diff
@@ -146,5 +152,193 @@ func processHostPathVolume(volume *api.HostPathVolume) (VolumeHostPathAndMode, e
 }
 
 func processGcePersistentDiskVolume(volume *api.GcePersistentDiskVolume) (VolumeHostPathAndMode, error) {
-	return VolumeHostPathAndMode{}, fmt.Errorf("Unimplemented.")
+	if volume.FsType != "" && volume.FsType != ext4FsType {
+		return VolumeHostPathAndMode{}, fmt.Errof("Unsupported filesystem type: %s", volume.FsType)
+	}
+	volume.FsType = ext4FsType
+	if volume.PdName == "" {
+		return VolumeHostPathAndMode{}, fmt.Errof("Empty PD name!")
+	}
+
+	devicePath, err := resolveGcePersistentDiskDevicePath(volume.Name)
+	if err != nil {
+		return VolumeHostPathAndMode{}, fmt.Errorf("Could not resolve GCE Persistent Disk device path: %s", err)
```

Comment:
What about being consistent with "PD" naming in the errors communicated to the user and either using "PD" or "Persistent Disk" or "GCE Persisten Disk" everywhere?

Reply:
I uniformized the errors.

Reply:
Thanks.